README.adoc

```diff
@@ -1034,6 +1034,7 @@ Some interesting links:
 
 * link:https://en.wikipedia.org/wiki/One-China_policy[] every country has private relations with Taiwan through some trade organizations, they just don't call them embassies
 * link:http://travel.stackexchange.com/questions/62447/how-can-a-citizen-of-mainland-china-visit-taiwan/62466#comment178495_62466[] hi, I want a _not VISA_ to Taiwan. Thanks.
+* link: https://youtu.be/6aZkM_ZLvjE?t=50[] Ko Wen-je 柯文哲, the mayor of Taipei 台北 and a professor in medicine, applies the concept of isolation and mutation in evolution to Taiwan and China.
```

Comment:
The whitespace on the right of the leftmost colon shouldn't be there.  My careless misake.